store/blogSlice.ts

```diff
@@ -10,41 +10,23 @@ import type { RootState } from './store';
 // Define a type for the slice state
 interface Blog {
   title: string;
-  creator: string;
-  updatedDatetime: string;
   coverImageUrl: string;
-  content: string;
-  categories: string[];
   featured: boolean;
-  description: string;
+  updatedDatetime: string;
+  author?: string;
+  content?: string;
+  description?: string;
+  tags:
+    | {
+        id: number;
+        Tag: string;
+      }[]
+    | [];
```

Comment:
isnt this the same thing or am i tripping

```suggestion
  tags: {
        id: number;
        Tag: string;
      }[];
```